lib/smart_answer_flows/landlord-immigration-check.rb

```diff
@@ -87,10 +87,13 @@ def define
 
       #Q3a
       multiple_choice :what_nationality? do
+        option "british-or-irish"
```

Comment:
Not sure if this is an issue. The node is not being exercised because you have not run the script:

```sh
rails r script/generate-responses-and-expected-results-for-smart-answer.rb --smart-answer-name--
```